README.md

```diff
@@ -47,6 +47,13 @@ case this plugin in present, use:
 Likewise, `category.shortname`, `category.parent` and `category.ancestors` can
 also be used on the category template.
 
+Additionally, this plugin adds `category.subcategories`: a `list` of categories
+that have `category` as a parent.
+
+    {% for subcat in category.subcategories %}
+        <a href="{{ SITEURL }}/{{subcat.url}}">{{subcat.shortname|capitalize}}</a>
+    {% endfor %}
+
```

Comment:
"`category.children` and `category.descendents`", and change the example to use `category.children`